tests/Microsoft.Fx.Portability.MetadataReader.Tests/ManagedMetadataReaderTests.cs

```diff
@@ -227,6 +220,23 @@ public void MultidimensionalPrimitiveArray()
             Assert.Equal(systemObject.DefinedInAssemblyIdentity, definedInAssemblyIdentity);
         }
 
+        [Fact]
+        public void ThrowsSystemObjectNotFoundException()
+        {
+            var dependencyFilter = Substitute.For<IDependencyFilter>();
+            dependencyFilter.IsFrameworkAssembly(Arg.Any<AssemblyReferenceInformation>()).Returns(false);
+            var dependencyFinder = new ReflectionMetadataDependencyFinder(dependencyFilter);
+            var assemblyToTest = TestAssembly.Create("FilterApis.cs", false, new[] { typeof(Image).GetTypeInfo().Assembly.Location });
```

Comment:
Why `System.Drawing.Image`? Doesn't that make this test not portable?

Reply:
True. I was trying to find a type that required an additional assembly reference. I'll fix this